python/examples/core/harness.py

```diff
@@ -26,7 +27,7 @@ def log(*args):
 
 
 def timed_invoke(run_n_iters: Callable, gflop_count: float, gbyte_count: float,
-                 n_iters: int):
+                 n_iters: int) -> Mapping[str, Any]:
```

Comment:
Nit: isn't the mapped type always a `List[float]` ?

Reply:
Improved signatures as requested.